index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="An open source re-implementation of Chris Sawyer's Locomotion."/>
+    <meta name="keywords" content="github, openloco, chris sawyer, locomotion, open source, tycoon, company, game"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oxygen&display=swap" rel="stylesheet">
+    <title>OpenLoco</title>
+</head>
+<body>
+    <div class="index">
+        <div class="bg-col">
+            <img class="logo-mobile" src="assets/image/47774002.png" alt="Logo">
```

Comment:
could you give the assets better names. We have a svg of the logo as well https://github.com/OpenLoco/OpenLoco/tree/master/resources/logo

Reply:
Will take a look and make adjustments. 👍